packages/block-editor/src/components/editor-fonts-resolver/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, useMemo } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { useGlobalSetting } from '../global-styles/hooks';
+import { getDisplaySrcFromFontFace, loadFontFaceInBrowser } from './utils';
+
+function EditorFontsResolver() {
+	const [ loadedFontUrls ] = useState( new Set() );
+
+	const { currentTheme } = useSelect( ( select ) => {
+		return {
+			currentTheme:
+				// Using 'core' as string to avoid circular dependency importing from @wordpress/core-data
+				select( 'core' ).getCurrentTheme(), // eslint-disable-line @wordpress/data-no-store-string-literals
```

Comment:
```suggestion
				// Disable Reason: Using 'core' as string to avoid circular dependency importing from @wordpress/core-data.
				// eslint-disable-next-line @wordpress/data-no-store-string-literals
				select( 'core' ).getCurrentTheme(),
```

This is just a slight formatting suggestion based on how I see this used elsewhere in the codebase.